docs/docs/whatsnew.md

```diff
@@ -41,3 +41,24 @@ If you want to use the old `mvn -B package` command instead of the `maven` build
 ```
 
 The same approach can be implemented if the `maven` builder is used in the global build step.
+
+## v1.2.25
+The build-parameters ignore attribute will be upgraded to support [Full Glob Pattern](https://en.wikipedia.org/wiki/Glob_(programming)). 
+
+Wildcards, such as `*`,`!`,`?` can be used to match characters. You can package specified content by using negation pattern `!`
```

Comment:
Wildcards, such as `*`,`!`,`?` can be used to match characters. You can package specific content by using the `!` negation pattern.
(Is there a mistake? How can it be that "!" is used to match and to negate?
Also, can we say "select specific content" in the second sentence above instead of "package specified content"? Is this correct?)

Reply:
By using negation pattern `!` in the `build-parameters ignore` attribute, because `not ignore` equal to `include`, you can select specified content into the package.